elementary/messages/formats/block_kit.py

```
@@ -257,24 +251,49 @@ def _add_divider_block(self, block: DividerBlock) -> None:
+        }
+
+    def _make_data_cell(self, value: Any) -> dict:
+        return {"type": "raw_text", "text": str(value) if value is not None else ""}
+
     def _add_table_block(self, block: TableBlock) -> None:
         column_count = len(block.headers)
-        if column_count not in self._MAX_CELL_LENGTH_BY_COLUMN_COUNT:
+
+        if column_count > self._MAX_SLACK_TABLE_COLUMNS or self._has_table_block:
             dicts = [
                 {header: cell for header, cell in zip(block.headers, row)}
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)